beekeeper-core/src/main/java/com/expediagroup/beekeeper/core/model/HousekeepingPath.java

```diff
@@ -50,9 +50,9 @@
   @Column(name = "table_name")
   private String tableName;
 
-  @Column(name = "path_status", nullable = false)
```

Comment:
So this is gonna require a migration script + documentation how to move from version 2 to 3.